tbtadm/controller.cpp

```diff
@@ -578,14 +578,16 @@ void tbtadm::Controller::approveAll(const fs::path& dir)
 // TODO: move to tbtadm-helper
 void tbtadm::Controller::approve(const fs::path& dir) try
 {
+    bool already_authorized = false;
+
     chdir(dir);
     m_out << "Authorizing " << dir << '\n';
 
     File authorized(authorizedFilename, File::Mode::Read);
```

Comment:
The check can be moved to after the adding to ACL, if the addition is done anyway.

Reply:
Sorry, which check?

Reply:
The check if the device is already authorized shouldn't be here but before trying to authorize it.
It was here so we return early without adding to ACL, but the whole point of this PR is to change this logic.